static/scss/partials/header.scss

```diff
@@ -86,6 +86,7 @@ header {
     padding: $spacing-xs $spacing-sm;
     color: $color-white;
     margin-left: $spacing-md;
+    font-size: 1rem;
```

Comment:
Removed `1.1rem` in the Protocol button overwrite, I found that just adding `1rem` standardizes the size for all of the header buttons. Thank you for that suggestion!